src/components/Hero/index.jsx

```diff
@@ -149,7 +150,7 @@ class Hero extends React.Component {
               {this.state.detailsOpen ? strings.hide_details : strings.show_details}
             </HeroDetailsButton>
           </HeroFooter>
-          {this.state.detailsOpen && <AttributesBlock hero={hero} />}
+          {this.state.detailsOpen && <AttributesBlock hero={compileLevelOneStats(hero)} />}
```

Comment:
I'm not sure but shouldn't this happen inside the AttributesBlock component since it's more of a "default behaviour" for stats in general?

Reply:
fixed